ompi/mca/coll/han/coll_han_alltoallv.c

```diff
@@ -194,31 +270,43 @@ static inline int alltoallv_sendrecv_w_direct_for_debugging(
                 jrecvs_completed++;
         }
 
-        requests[jreq] = &ompi_request_null.request;
         if (ii_send_req && jsends_posted < ntypes_send) {
             rc = ompi_datatype_create_contiguous( 1, (ompi_datatype_t*)send_types[jsends_posted], &yuck_ompi_dtype_from_opal );
```

Comment:
you really called this datatype yuck !? It must be an acronym right ?

Reply:
I had to synthesize it from an opal datatype on-demand so that I could pass it to sendrecv.  I can rename it.

Reply:
Renamed.